components/atom/backToTop/UXDEF.md

```diff
@@ -0,0 +1,126 @@
+# SUI - Back to Top
+A simple way to return back to the top of a long content page.
+
+|   Status          | WIP |
+|   :----           |   :---- |
+|   Current version |   1.0|
+|   Category        |   Atom |
+|   Owners          |   UX @Irene L, @Chris J |
+
+## Structure
+
+Back to Top consists in a floating element located on the bottom right corner of the page and with a [z-index](https://paper.dropbox.com/doc/z-index-WIP--AMqq1DAVVk5QOOeNOojmYuRrAg-4QvLsXUoWXeWNDiQSzCBc) position.
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_2B6461E0A5AACA052E49BE35E96B0E3C48FC369AADACF7054C77A66FA69B339E_1536830620353_1-+Back+to+Top+-+Structure.png)
+
+## Behavior
+
+Back to Top shows up when the user scrolls down beyond 50% of the visible part of the page and disappears when the user reaches that same limit.
+When the user clicks or taps on the Back to Top, the page scrolls to the top of the page and the Back to Top disappears.
+
+
+## Types
+
+Back to Top has two **contrast variations** and three **content options** that can be combined as needed.
+
+**Contrast variations** in order to keep an optimal recognition:
+
+- For light background pages, use a a dark background Back to Top.
+- For dark background pages, use a light background Back to Top.
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_2B6461E0A5AACA052E49BE35E96B0E3C48FC369AADACF7054C77A66FA69B339E_1536830638907_2.1-+Back+to+Top+-+Types+-+Contrast.png)
+
+**Content options** in order to adapt to different needs and device contexts:
+
+- Icon (up arrow) + text label.
+- Only icon (up arrow).
+- Only text label.
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_2B6461E0A5AACA052E49BE35E96B0E3C48FC369AADACF7054C77A66FA69B339E_1536830649012_2.2-+Back+to+Top+-+Types+-+Content.png)
+
+## Contents
+
+- The icon should always be an arrow pointing up.
+- The text label should be short and brief, preferably one word (Top, Up, Subir, Ir arriba, etc.)
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_2B6461E0A5AACA052E49BE35E96B0E3C48FC369AADACF7054C77A66FA69B339E_1536847244254_3-+Back+to+Top+-+Content.png)
+
+## Visual
+
+Visuals should be custom on each vertical. This specs are for a SUi white label design:
+
+FONT
+font-size: #fs-m (font-size:16px & line-height:20px) 
+
+ICON
+size: #m 
+
+COLORS DARK CONTRAST
+background $bgc-gray-d2 
```

Comment:
I don't know if it would be better to write some kind of metalanguage instead of using the real Sass variables names. What do you think about?

Reply:
yeap, not only colours but the same with all vars

Reply:
+1